breakout_3_comments.md

```diff
@@ -0,0 +1,16 @@
+# Reviewer Comments
+## Reviewer 1
+### General comments
+I thought this figure looked very good before? I'm not sure what silly reviewer wanted changes done to it, but it doesn't look good anymore!The colors are absurd! I have listed my comments on what to revert or change.
+
+### Specific comments
+- Revert the changes done to the peaches color to its original color. **_Refer to script 1_**
+- Revert the changes done to the leafs color to its original color. **_Refer to script 2_**
+- The text for 'Virtual 2020' is to large and overlapping the line. Reduce the text size to 16. **_Refer to script 5_**
```

Comment:
I only see 3 changes for this breakout. What are the other group members supposed to do? Each script should have a commit to revert back to so everyone practices this. Then, you can give some an additional change to make it fancier. 

We don't want someone to just go "oh I remember what the RGB was, I'll type it in". We want them to actually revert back to a particular commit. To encourage this behavior, the intro paragraph for this handout can say something like "After months of waiting to hear back from Nature...", implying that it's been so long since you made the last changes that you don't remember and should just revert to the commit. Hopefully their commit message from before is informative! Walk them through the process I outlined for you here: https://github.com/alcantarar/ASB_Tutorial/issues/3#issuecomment-655848607

Reply:
I think leaving them with only 3 changes is fine, as reverting or checking out to previous commits is a little more complicated than the other stuff they've done.

Reply:
But there are 5 people in a group and 3 things to change. What are the other group members supposed to do?